.github/workflows/qa.yaml

```diff
@@ -15,8 +15,7 @@ jobs:
   test:
     uses: canonical/starflow/.github/workflows/test-python.yaml@main
     with:
-      # Limiting to amd64 as a workaround for https://github.com/canonical/craft-archives/issues/144
-      # When fixing that bug, add arm64 runners as well for fast tests.
-      fast-test-platforms: '[["jammy", "amd64"], ["noble", "amd64"]]'
-      slow-test-platforms: '[["jammy", "amd64"]]'
-      lowest-python-platform: self-hosted-linux-amd64-jammy-edge
+      # Testing on both amd64 and arm64 to prevent architecture-specific bugs.
+      fast-test-platforms: '["jammy", ["noble", "amd64"], ["noble", "arm64"]]'
```

Comment:
Is that intentional to not also explicitly set arm64 and amd64 for jammy here?

Reply:
Yes. That way we run on both architectures on noble, but we just check whatever's available first on jammy.